src/modules/Dashboard/html_client/mod_dashboard_index.html.twig

```diff
@@ -1,6 +1,14 @@
 {% extends request.ajax ? "layout_blank.html.twig" : "layout_default.html.twig" %}
 
-{% block meta_title %}{{ 'Client Area'|trans }}{% endblock %}
+{% block meta_title %}
+{% if client %}
+    {{ 'Client Area'|trans }}
```

Comment:
So in what cases is "Client area" still used? 

Why not remove it at all?

Reply:
It is used when a client is logged in

Reply:
Don't want we to have it as well "Company name  | Client area" instead

Reply:
Or even keep it as well as "Company name"

Reply:
It has a advantage to have at least the company name in the title  (Or tabs at all time)

Reply:
> Don't want we to have it as well "Company name | Client area" instead

This makes a lot of sense to me.

Reply:
I think the best option would be letting the administrator choose. Just like the `$ {{price}}` thing we do in currencies, we can implement something similar: `{{title}} | {{company_name}}`

Reply:
Or Client Area - Company is fine for me aswell